polymetis/polymetis/python/scripts/robopen_dashboard_demos/square_movement.py

```diff
@@ -0,0 +1,110 @@
+import a0
+import polymetis
+import torch
+import time
+import serdes
+import json
+
+if __name__ == "__main__":
+    std_msgs = serdes.get_capnp_msgs("std_msgs")
+    geometry_msgs = serdes.get_capnp_msgs("geometry_msgs")
+    transform_pub = a0.Publisher("robot/desired_ee_pose/square")
+    json_pub = a0.Publisher("robot/desired_ee_pose/square_json")
+
+    robot = polymetis.RobotInterface(ip_address="100.97.47.78", enforce_version=False)
+    robot.go_home()
+
+    ee_pos, ee_quat = robot.get_ee_pose()
+
+    # Move to starting position
+    starting_position = -0.1
+    ee_pos += torch.Tensor([0.0, starting_position, starting_position])
+    robot.move_to_ee_pose(ee_pos)
+
+    y_ee_limit = 0.45
+    z_ee_limit = 0.7
+    side_length = min(y_ee_limit - ee_pos[1], z_ee_limit - ee_pos[2])
+
+    # Square using move_to_ee_pose
+    # for i in range(4):
+    #     if i == 0:
+    #         ee_pos += torch.Tensor([0.0, side_length, 0.0])
+    #     elif i == 1:
+    #         ee_pos += torch.Tensor([0.0, 0.0, side_length])
+    #     elif i == 2:
+    #         ee_pos += torch.Tensor([0.0, -side_length, 0.0])
+    #     else:
+    #         ee_pos += torch.Tensor([0.0, 0.0, -side_length])
+
+    #     robot.move_to_ee_pose(ee_pos)
+
+    # Square using update_desired_ee_pose
+    control_hz = 50
+    time_left = 10
+    steps_per_side = control_hz * time_left / 4
+    ee_step = side_length / steps_per_side
```

Comment:
Same comment regarding commented out code as above.

Reply:
Thanks! I decided instead to just remove it since it creates a square that pauses on the corners and this section doesn't include pubbing the messages to the dashboard.